relay-server/src/actors/store.rs

```diff
@@ -744,7 +819,7 @@ impl Service for StoreService {
 }
 
 /// Common attributes for both standalone attachments and processing-relevant attachments.
-#[derive(Debug, Serialize)]
+#[derive(Debug, Serialize, Clone)]
```

Comment:
This `Clone` is not necessary.
 ```suggestion
#[derive(Debug, Serialize)]
```